dotnet/src/Connectors/Connectors.OpenAI/AudioToText/OpenAIAudioToTextService.cs

```diff
@@ -66,6 +67,50 @@ public sealed class OpenAIAudioToTextService : IAudioToTextService
         this._core.AddAttribute(AIServiceExtensions.ModelIdKey, modelId);
     }
 
+    /// <summary>
+    /// Creates an instance of the <see cref="OpenAIAudioToTextService"/> for Custom Audio OpenAI API
+    /// </summary>
+    /// <param name="modelId">Model name</param>
+    /// <param name="endpoint">Custom OpenAI Audio Transcription API compatible endpoint</param>
+    /// <param name="apiKey">OpenAI API Key</param>
+    /// <param name="organization">OpenAI Organization Id (usually optional)</param>
+    /// <param name="httpClient">Custom <see cref="HttpClient"/> for HTTP requests.</param>
+    /// <param name="loggerFactory">The <see cref="ILoggerFactory"/> to use for logging. If null, no logging will be performed.</param>
+    public OpenAIAudioToTextService(
```

Comment:
Add [Experimental] SK0010 for all methods introduced in this PR, 

Keep in mind we might change or adapt them in the next v2 connector version.

Reply:
> Keep in mind we might change or adapt them in the next v2 connector version.

Does it make sense to wait until v2 connection version will be ready and build on top of it rather than do it now and change it later?

Reply:
Agree, late next week we might be already with V2 on main.